src/hybrid/default-url-mapping-table.ts

```diff
@@ -30,6 +30,11 @@ export const ICM_WEB_URL = '/INTERSHOP/web/WFS/$<channel>/$<lang>/$<application>
 
 /**
  * Mapping table for running PWA and ICM in parallel
+ *
+ * NOTE:
+ * THIS MAPPING TABLE IS JUST AN EXAMPLE IMPLEMENTATION.
+ * IT DOES NOT DEFINE MAPPINGS FOR ALL ROUTES THAT COULD BE HANDLED IN THE PWA OR THE ICM.
+ * IT NEEDS TO BE ADAPTED TO THE NEEDS OF THE PARTICULAR PROJECT.
```

Comment:
Maybe the last two lines shouldn't be shouting at developers.

Reply:
Yes, maybe not.

Reply:
I stopped the continuing shouting.